ir/v1.def

```diff
@@ -252,8 +252,15 @@ class CalculatedField {
         v.visit(annotations, "annotations"); }
 }
 
+class ParserValueSet {
+    optional ID                 name;
+    Annotations                 annotations;
+    dbprint { out << node_type_name() << " " << name; }
+    toString { return node_type_name() + " " + name; }
+}
+
 class CaseEntry {
-    safe_vector<std::pair<Constant, Constant>>  values = {};
+    safe_vector<std::pair<Literal, Constant>>    values = {};
```

Comment:
This should be a `safe_vector<Expression>`, though that will require more changes in everything that visits a CaseEntry (the Expression must be a Constant, a Mask, or (now) a PathExpression).  The P4_16 SelectCase already has an Expression here, so it should actually be simpler to deal with.

Reply:
Alternately, to keep the change smaller, could be `safe_vector<std::pair<Expression, Constant>>`, keeping the mask split off.